app-web/src/pages/index.js

```diff
@@ -55,33 +56,73 @@ const getUniqueResources = resources => {
   let events = resources.filter(resource => resource.resource.type === RESOURCE_TYPES.EVENTS);
   let allButEvents = resources.filter(resource => resource.resource.type !== RESOURCE_TYPES.EVENTS);
   allButEvents = uniqBy(allButEvents, 'unfurl.title');
-  return allButEvents.slice(0, 15).concat(events);
+  return allButEvents.concat(events);
+};
+
+/**
+ * returns the text and path being used for the link in the ResourcePreview below
+ * the text will be reflective of the resourceType and is sensitive to different search result # cases
+ * the link takes you to the corresponding resource type page but with the search active on that page
+ */
+const getTextAndLink = (resourceType, resourcesByType) => {
+  // The resourceSearchPath will give you the string of the query as its used in the URL
+  // ex: '?q=Open%20Shift' so that we can use it in the link for each resourceType result
+  // This opperates under the assumption that there will be 3 occurences of '/' before such a string
+  let resourceSearchPath = window.location.href.split('/');
```

Comment:
i'd replace this with `window.location.search` so that you do need to work under any assumptions